.redhat-ci.yml

```diff
@@ -1,12 +1,9 @@
 host:
-    distro: fedora/24/cloud
-branches:
-    - master
+    distro: fedora/24/atomic
 packages:
```

Comment:
Right, that's correct. Probably easier to just maintain a `test.sh` script in your repo. Will work on a `containerized: true` type of thing to make this more seamless in the future.